src/vcpkg/vcpkgcmdarguments.cpp

```diff
@@ -636,6 +636,23 @@ namespace vcpkg
         msg::println(LocalizedString::from_raw(table.m_str));
     }
 
+    // Merges overlay ports in the specific order
+    void VcpkgCmdArguments::merge_overlays() {
+        overlay_ports.insert(
+            std::end(overlay_ports), std::begin(manifest_overlay_ports), std::end(manifest_overlay_ports));
+        overlay_ports.insert(
+            std::end(overlay_ports), std::begin(env_overlay_ports), std::end(env_overlay_ports));
+        overlay_triplets.insert(
+            std::end(overlay_triplets), std::begin(manifest_overlay_triplets), std::end(manifest_overlay_triplets));
+        overlay_triplets.insert(
+            std::end(overlay_triplets), std::begin(env_overlay_triplets), std::end(env_overlay_triplets));
```

Comment:
the order defined here needs documentation

Reply:
I have added the PR to update the docs in the description of this PR